### PR TITLE
enable disaster dashboard for prod and staging

### PR DESCRIPTION
**Pull request by chejennifer**
- enable disaster dashboard and event page (because disaster dashboard links to event pages)
- add disaster dashboard to explore dropdown
- add webdriver test for event page

### CODE REVIEW

File: server/app_env/production.py [hunk 1]
```diff
@@ -16,4 +16,5 @@
 
 
 class Config(_base.Config):
-  GA_ACCOUNT = 'UA-117119267-1'
\ No newline at end of file
+  GA_ACCOUNT = 'UA-117119267-1'
+  SHOW_DISASTER = True
```
Comment (shifucun) [inline server/app_env/production.py:20]: Do we want to make it "True" by default?

File: server/__init__.py [hunk 2]
```diff
@@ -98,30 +98,33 @@ def register_routes_custom_dc(app):
   pass
 
 
-def register_routes_disasters(app):
-  # Install blueprints specific to Stanford DC
+def register_routes_disasters(app, include_sustainability):
```
Comment (shifucun) [inline server/__init__.py:101]: why have "include_sustainability" here? I feel register_xxx() should be a simple function. All the condition check should be outside and, if needed, have another register_xxx() to hold other related logic groups.

Comment (shifucun) [reply server/__init__.py:101]: if sustainability is different from disaster. then can use SHOW_SUSTAINABILITY flag in the env py files. We should try to remove the if/else checks here as much as possible..

File: server/app_env/production.py (hunk 1 above)
Comment (chejennifer) [reply server/app_env/production.py]: done

**Review by shifucun — APPROVED** [review]
Is "ENABLE_DISASTER_JSON" still needed in local? If not, maybe remove it.

**Comment by chejennifer** [comment]
> Is "ENABLE_DISASTER_JSON" still needed in local? If not, maybe remove it.

we still have the ability to use the disaster json for the disaster dashboard, so we should probably keep that flag until we decide to remove that option.. And probably want to do in separate PR

**Pull request merged**